Document the RelationManager FormWidget (#131)
wintercms/docs: backend-forms.md

- [Nested Form](#widget-nestedform)
- [Record finder](#widget-recordfinder)
- [Relation](#widget-relation)

> - [Relation manager](#widget-relationmanager)

Reviewer: ```suggestion
- [Relation Manager](#widget-relationmanager)
```

`emptyOption` | text to display when there is no available selections.
`scope` | specifies a [query scope method](../database/model#query-scopes) defined in the **related form model** to apply to the list query always.

<a name="widget-relationmanager"></a>
### Relation Manager

> `relationmanager` - Renders the relation controller for this relation. This is the equivalent of using a partial field type pointing to a partial file that will call the relationRender() method on the controller.

Reviewer: ```suggestion
`relationmanager` - Renders the [relation controller](../relations) for this relation. This is the equivalent of using a `partial` field that renders the output of the `relationRender()` method from the controller.
```
